apps/web/components/availability/ScheduleListItem.tsx

```diff
@@ -0,0 +1,69 @@
+import { DotsHorizontalIcon, TrashIcon } from "@heroicons/react/solid";
+import Link from "next/link";
+import { Fragment, useState } from "react";
+
+import { availabilityAsString } from "@calcom/lib/availability";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Availability } from "@calcom/prisma/client";
+import { Button } from "@calcom/ui";
+import Dropdown, { DropdownMenuContent, DropdownMenuItem, DropdownMenuTrigger } from "@calcom/ui/Dropdown";
+
+import Loader from "@components/Loader";
+
+export function ScheduleListItem({ schedule, deleteFunction }: { schedule: any; deleteFunction: Function }) {
```

Comment:
We try avoid `any`s unless strictly necessary.

Reply:
I didn't find the correct type at the moment so I use `any`. I saw you already committed an correction.